command/rm.go

```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/peak/s5cmd/storage/url"
 )
 
+const (
```

Comment:
I know this is intended for reusing error messages in tests. However, I would prefer to rewrite them in tests rather than declaring global variables.

Reply:
hmm, thanks :)

Reply:
I don't have any special reasoning for this :)  I might be wrong but we should be consistent while handling errors. Since other commands don't have global error messages,  I thought these error messages should not be global as well. Either way is okay for me.